### DISCUSSION TITLE
AllSky Camera Housing/Mount

### DISCUSSION THREAD

**Discussion by RoySalisbury** [aaronwmorris/indi-allsky#669] [Show and tell]
I built my first AllSky Camera about 4 years ago.  At the time it was using my own software to control my ASI174MM using C# on RPi 3 (later RPi 4).  I eventually moved to the original allsky software, and later the indi-allsky version (it has active development).

 I designed the housing in Fusion 360 and printed the necessary parts on my 3D printer (the rest I purchased). Its been working since day one.

Eventually the camera dome got a bit hazy due to the sun and the only supplier of the size that I was using is not longer stocking them.  So I had to find a new dome.  That also meant redesigning my camera enclosure. So I sepnt the day yesterday in Fusion 360  and came up with an easy to print / assembly enclosure.  Here is a 3D rendering:

![AllSkyCamera Enclosure v4](https://user-images.githubusercontent.com/12501943/229389713-46c5110d-213a-4c8a-a445-f7cdc48f7a89.png)

Here is a cutaway view to see the parts inside:

![AllSkyCamera Enclosure v4 - CutAway](https://user-images.githubusercontent.com/12501943/229389843-2afa4a1f-8c1e-49fd-b530-a180b3667aa5.png)

And here is a stack up of the hardware (I had to create 3D models of each component)

![AllSkyCamera Enclosure v4 - Hardware](https://user-images.githubusercontent.com/12501943/229390068-44a1ac3b-a13b-492f-9f83-d98cb67ee3a6.png)

Initially the lower section was a PVC 4" to 1.5" reducer from Home Depot, but I found I could 3D print it cheaper.  The Camera in this unit is the ASI 178MC, but it also works with my ASI174MM.  The idea is that the housing will attach to a PVC pip or other tubular mount and the USB cable will route out of that into a box for the Raspberry Pi.

If anyone is intreated in the 3D printer files and a parts list, let me know.  I printed it in ASA so it is UV and temperature resistant (been holding up for 4 years on the last print).

I've also started designing one up for the ASI174MM Mini (any of the ASI mini cameras would work).  Still need some work on it, but its a start ... and very small (smaller lens and dome). I could also use the larger camera lens (a better one) in the same size enclosure).

![AllSky Mini v2](https://user-images.githubusercontent.com/12501943/229392497-7c286fb2-cf11-470f-b7b4-c53b7b766276.png)

**Comment by aaronwmorris** [comment]
Interesting.  What lens is that, it seems to be much longer than I am used to seeing for an all sky lens.  I am guessing that is a 3" dome?

**Reply by RoySalisbury** [reply]
The lens is a Fujinon FE185C086HA-1 1" C-Mount 2.7mm F/1.8

https://www.amazon.com/Fujinon-FE185C086HA-1-C-Mount-2-7mm-1-8/dp/B00HSY339Q

They also have 2 other models for different sensor sizes.  I choose it because it has great optical quality and little to no distortions across the entire field of view.

The dome is a SUPREMETECH Acrylic Dome/Plastic Hemisphere - Clear - 2.5" Diameter, 3/8 Flange

https://www.amazon.com/dp/B07CVPTP4D
